### PR TITLE
New tsbs

### PR DESCRIPTION
**Pull request by raidoz**

### CODE REVIEW

File: make/boards/tsb1.board [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+UUID_BOARD          := b1f61ed1-bc69-4850-9d09-1f388f554653
+
+# Maximum size limits for binaries in memory, their start addresses
+BOOTLOADER_START    := 0x00000
+BOOTLOADER_MAX_LEN  := 0x14000
+STAGE1_START        := 0x00000
+STAGE1_MAX_LEN      := 0x0C000
+STAGE2_START        := 0x0C000
+STAGE2_MAX_LEN      := 0x08000
+APP_START           ?= 0x14000
```
Comment (kobilozor) [inline make/boards/tsb1.board:61]: Shall we have APP_START as 0 for tsb platform?

File: make/boards/tsb2.board [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+UUID_BOARD          := c359a127-46a0-47f2-b827-2c21e284bd59
+
+# Maximum size limits for binaries in memory, their start addresses
+BOOTLOADER_START    := 0x00000
+BOOTLOADER_MAX_LEN  := 0x14000 # 64K+32K
+STAGE1_START        := 0x00000
+STAGE1_MAX_LEN      := 0x10000 # 64K
+STAGE2_START        := 0x10000
+STAGE2_MAX_LEN      := 0x08000 # 32K
+APP_START           ?= 0x18000 #
```
Comment (kobilozor) [inline make/boards/tsb2.board:61]: Shall we have APP_START as 0 for tsb platform?

File: make/boards/tsb1.board (hunk 1 above)
Comment (raidoz) [reply make/boards/tsb1.board:61]: Not necessarily, will want bootloader in some scenarios also.

File: make/boards/tsb2.board (hunk 2 above)
Comment (raidoz) [reply make/boards/tsb2.board:61]: Not necessarily, will want bootloader in some scenarios also.

**Review by kobilozor — APPROVED** [review]

**Pull request merged**